### DISCUSSION TITLE
Welcome to react-native-qrcode-scanner discussions!

### DISCUSSION THREAD

**Discussion by moaazsidat** [moaazsidat/react-native-qrcode-scanner#301] [General]
## 👋 Hey everyone!

I've enabled Github Discussions on this repo to facilitate asking questions and getting response from the community. I know some of you have reached out via DMs or email, but haven't gotten a reply from me in a timely manner, mostly due to my own time constraints. 

I think we can start using discussions to:
  * Ask questions
  * Share ideas
  * Engage with community memebers

Please remember to be respectful to everyone! Thanks! 

If you've seen some aspects of Discussions work well in other repos, reply down below, would love to learn!

**Comment by mbliok** [comment]
Hello guys,
I checkout the project you did. Thank you.
My question is: Why the Android and Ios  folders are missing?

Please explain to me, I'm new to react-native
